index.html

```diff
@@ -63,8 +59,15 @@
     <div class="row bottom-space">
       <h2>Workshop</h2>
       <div class="col-md-6">
-        <p><b>English: </b><a href="http://www.clojurebridge.org/">ClojureBridge</a> aims to increase diversity within the Clojure community by offering free, beginner-friendly Clojure programming workshops for women.</p>
-        <p><a href="http://clojure.org/">Clojure</a> is a modern programming language. We will use it to introduce you to fundamental programming concepts and build a creative app.</p>
+        <p><b>English: </b><a href="http://www.clojurebridge.org/">ClojureBridge</a>
+          aims to increase diversity within the Clojure community by offering free,
+          beginner-friendly Clojure programming workshops for people who
+          identify as <b>woman or a non-binary gender</b>.
```

Comment:
Just "or non-binary" should work well here, as it's a designation (albeit a meta one) on its own, and not just an adjective